src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/Cohort.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+using SFA.DAS.Authorization;
+
+namespace SFA.DAS.ProviderCommitments.Web.Models
+{
+    public class Cohort : IAuthorizationContextModel
+    {
+        [Required]
```

Comment:
It looks like fluent validation has been included in the Mvc pipeline so the validation attributes shouldn't be needed here.

Reply:
Replaced with Fluent validation